ansible-collection/main.py

```diff
@@ -0,0 +1,519 @@
+import base64
+import json
+import os
+import re
+import subprocess
+import time
+from collections import defaultdict
+
+import yaml
+from kubernetes import client, config
+
+
+def get_failed_volumes(file_path):
```

Comment:
As get_failed_volumes and get_backup_name, operates on kubeconfig file. Can we pass the file as input ?

Reply:
Actually I need to remove the function `get_backup_name` it is not required and not used anywhere

Reply:
Fixed it